JASP-Engine/JASP/R/summarystatsbinomialtestbayesian.R

```diff
@@ -242,19 +242,19 @@ SummaryStatsBinomialTestBayesian <- function(jaspResults, dataset = NULL, option
     
     hypothesis_for_common_functions   <- "twoSided"
     hypothesis                        <- "two.sided"
-    message <- paste0("Proportions tested against value: ", theta0, ".")
+    message <- paste0(gettext("Proportions tested against value: "), theta0, ".")
     
   } else if (hypothesis_option == "greaterThanTestValue") {
     
     hypothesis_for_common_functions   <- "plusSided"
     hypothesis                        <- "greater"
-    message <- paste0("For all tests, the alternative hypothesis specifies that the proportion is greater than ", theta0, ".")
+    message <- paste0(gettext("For all tests, the alternative hypothesis specifies that the proportion is greater than "), theta0, ".")
     
   } else if (hypothesis_option == "lessThanTestValue") {
     
     hypothesis_for_common_functions   <- "minSided"
     hypothesis                        <- "less"
-    message <- paste0("For all tests, the alternative hypothesis specifies that the proportion is less than ", theta0, ".")
+    message <- paste0(gettext("For all tests, the alternative hypothesis specifies that the proportion is less than "), theta0, ".")
```

Comment:
`gettextf("For all tests, the alternative hypothesis specifies that the proportion is less than %i.", theta0)`

Reply:
Done